### ISSUE TITLE
IP and port are configured but links are different

### ISSUE THREAD

**Issue by EverythingSuckz** [EverythingSuckz/TG-FileStreamBot#124]
Tkank you.But I found a new problem, the configuration in fsb.env is not taking effect. I customized the ip and port and the program starts with "Server started {"port": 9527}", but the link given is still http://localhost:8080/*.

_Originally posted by @holll in https://github.com/EverythingSuckz/TG-FileStreamBot/issues/122#issuecomment-1837476518_

**Comment by EverythingSuckz** [comment]
@holll You need to add `HOST` variable in your env file and that will be formatted in your links

add
```env
HOST=http://yourIP:port
```

See [fsb.sample.env](https://github.com/EverythingSuckz/TG-FileStreamBot/blob/golang/fsb.sample.env) for more info

**Comment by holll** [comment]
But I think it would be better not to add an additional port number to HOST when PORT is filled in?

**Comment by holll** [comment]
Please ignore my question, maybe I haven’t considered it comprehensively enough.

**Comment by EverythingSuckz** [comment]
> But I think it would be better not to add an additional port number to HOST when PORT is filled in?

You are right but I've given the user total choice to use any domain they want. If you want to set up a domain, it'll be way easier to just change the `HOST` rather than changing too many unnecessary configs like in the case of the [python branch](https://github.com/EverythingSuckz/TG-FileStreamBot#optional-vars).
